Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -377,9 +377,9 @@ bool CBoot::SetupWiiMemory(Core::System& system, IOS::HLE::IOSC::ConsoleType con
                                    IOS::HLE::FS::Mode::Read);
     if (file && file->Read(data.data(), data.size()))
     {
-      gen.SetBytes(data);
-      serno = gen.GetValue("SERNO");
-      model = gen.GetValue("MODEL");
+      Common::SettingsHandler settings_reader(data);
+      serno = settings_reader.GetValue("SERNO");
+      model = settings_reader.GetValue("MODEL");
```

Comment:
Please delete the `gen` variable on line 366.

Reply:
Moved the declaration of `gen` down to line 424 near its first use. (We still use that variable, but not until later in the method.)